cmd/lifecycle/creator.go

```diff
@@ -172,7 +172,8 @@ func (c *createCmd) Exec() error {
 			lifecycle.NewConfigHandler(),
 			dirStore,
 		)
-		detector, err := detectorFactory.NewDetector(c.AppDir, c.BuildConfigDir, c.OrderPath, c.PlatformDir, cmd.DefaultLogger)
+		// QUESTION:  does everyone else think it's correct to pass in an empty/default analyzed.MD here since in this old API version the detect runs before the analyze?
```

Comment:
Seems okay to me... this whole "factory.NewPhase" thing is crying out for a refactor (now that we have `LifecycleInputs` for all phases why not pass the inputs in as you suggested earlier instead of having N arguments... then maybe the creator could introduce some overrides for structs that we don't actually write out in a single-phase invocation - analyzed.toml, group.toml, plan.toml ...but let's get this PR through first).